### PR TITLE
[dv/kmac] Map sec_cm with current test

### PR DESCRIPTION
**Pull request by cindychip**
This PR maps the sec_cm testplan with the current existing test.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

File: hw/ip/kmac/data/kmac_sec_cm_testplan.hjson [hunk 1]
```diff
@@ -27,61 +27,61 @@
...
       stage: V2S
-      tests: []
+      tests: ["kmac_sec_cm"]
     }
     {
       name: sec_cm_cfg_shadowed_config_regwen
       desc: "Verify the countermeasure(s) CFG_SHADOWED.CONFIG.REGWEN."
       stage: V2S
-      tests: []
+      tests: ["{variant}_smoke"]
```
Comment (jdonjdon) [inline hw/ip/kmac/data/kmac_sec_cm_testplan.hjson:78]: I assume description will be updated in another PR right?

**Review by jdonjdon — APPROVED** [review]
LGTM with one comment.

**Review by matutem — APPROVED** [review]
LGTM
